pkg/handler/sam/emailContact/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
```

Comment:
What does the SAM package mean?

Reply:
it's just a name to specify that this package is for AWS SAM, maybe naming it `AWS` would be more suitable.